01week/datatypes.js

```diff
@@ -0,0 +1,80 @@
+// 1. Write a JavaScript program to display the current day and time.
+
+// Notes: method "Date()" can pull the current date/time
+
+const currentTime = Date();
+console.log(currentTime)
+
+
+// 2. Write a JavaScript program to convert a number to a string.
+
+// Notes: method "String()" can convert numbers to a string.
+
+const value2 = 100;
+const numberToString = String(value2)
```

Comment:
This is a variable that stores '100'. I need a function to turn any number into a string and return that string.